src/main/lib/file-utils.js

```diff
@@ -17,28 +20,54 @@ const readJson = (filename, processFn) => {
 const writeJson = (filename, json) => {
   return new Promise((resolve) => {
     fs.writeFile(filename, json, (err) => {
-      if (err) throw err;
+      if (err) {
+        throw err;
+      }
       resolve();
     });
   });
 };
 
-const exists = (path) => {
-  return fs.existsSync(path);
-};
+const exists = (path) => fs.existsSync(path);
 
-const listFilesInDirectory = (dir, excludes = []) => {
-  return fs.readdirSync(dir, { withFileTypes: true })
-    .filter((file) => {
-      if (file.isDirectory()) {
-        return true;
-      } else {
-        return path.extname(file.name) === '.json';
-      }
-    })
-    .filter((file) =>
-      !excludes.some(el => matcher.isMatch(file.name, el))
-    );
+const getJsonFiles = (directory, exclusions = []) => {
+  let paths = glob.sync(directory + '/**/*.json');
+  let relativePaths = toRelativePaths(paths, directory);
+  exclusions = exclusions.map(exclusion => exclusionUtils.prepareExclusion(exclusion));
+  let filteredPaths = exclusionUtils.filterPaths(relativePaths, exclusions);
+  return groupToSheets(filteredPaths);
 };
 
-module.exports = { writeJson, readJson, listFilesInDirectory, exists };
+function toRelativePaths (array, root) {
+  return array.map(
+    file => path.relative(root, file));
+}
+
+function groupToSheets (paths) {
```

Comment:
IMO it should not be there as it breaks abstraction level. File utils should be focused on file operations and know nothing about sheets.